packages/modules/src/modules/epics/ContributionsEpicChoiceCards.tsx

```diff
@@ -79,16 +114,68 @@ export const ContributionsEpicChoiceCards: React.FC<EpicChoiceCardProps> = ({
         trackClick('frequency');
         setSelectionsCallback({
             frequency: frequency,
-            amount: amountsForCountryGroup[frequency]['amounts'][1],
+            amount: amountsTestVariant[frequency].defaultAmount,
         });
     };
 
-    const frequencySuffix = () => {
-        return {
-            ONE_OFF: '',
-            MONTHLY: ' per month',
-            ANNUAL: ' per year',
-        }[selection.frequency];
+    const ChoiceCardAmount = ({ amount }: { amount?: number }) => {
+        if (amount) {
+            return (
+                <ChoiceCard
+                    value={`${amount}`}
+                    label={`${currencySymbol}${amount} ${
+                        contributionType[selection.frequency].suffix
+                    }`}
+                    id={`${amount}`}
+                    checked={selection.amount === amount}
+                    onChange={() => updateAmount(amount)}
+                />
+            );
+        }
+        return null;
+    };
+
+    const generateChoiceCardAmountsButtons = () => {
+        const productData = amountsTestVariant[selection.frequency];
+        const requiredAmounts = productData.amounts;
+        const hideChooseYourAmount = productData.hideChooseYourAmount ?? false;
+        const defaultAmount = amountsTestVariant[selection.frequency].defaultAmount;
+        console.log(defaultAmount, requiredAmounts);
+
+        // Something is wrong with the data
+        if (!Array.isArray(requiredAmounts) || !requiredAmounts.length) {
+            return <ChoiceCard value="third" label="Other" id="third" checked={true} />;
+        }
+
+        return (
+            <>
+                <ChoiceCardAmount amount={requiredAmounts[0]} />
+                <ChoiceCardAmount amount={requiredAmounts[1]} />
+                {hideChooseYourAmount ? (
+                    <ChoiceCardAmount amount={requiredAmounts[2]} />
+                ) : (
+                    <ChoiceCard
+                        value="other"
+                        label="Other"
+                        id="other"
+                        checked={selection.amount == 'other'}
+                        onChange={() => updateAmount('other')}
+                    />
+                )}
+            </>
+        );
+    };
+
+    const generateChoiceCardFrequencyTab = (frequency: ContributionFrequency) => {
+        return (
+            <ChoiceCard
+                label={contributionType[frequency].label}
+                value={contributionType[frequency].value}
```

Comment:
as far as I can tell `value` isn't actually used, can we just use `frequency` instead?

Reply:
It's s Source component and (as far as I can tell) a required attribute for the component[1]. I've got no idea why it's required or how it's used[2].

That said, I __think__ `value` is only used in this one place. If we can be happy that `frequency` is a good-enough substitute, then I'd be more than happy to get rid of it, both here and in the `contributionsType` static data object I've defined on line 57.

[1] - https://guardian.github.io/csnx/?path=/story/source-react-components_choicecard--default-default-theme
[2] - https://github.com/guardian/csnx/blob/main/libs/%40guardian/source-react-components/src/choice-card/ChoiceCard.tsx

Reply:
yes that's what I meant, you can just use frequency everywhere instead

Reply:
I'll trust your judgment on this one and get rid of `value`

Reply:
... Done!